modules/nf-core/kaiju/mergeoutputs/main.nf

```diff
@@ -0,0 +1,51 @@
+process KAIJU_MERGEOUTPUTS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kaiju:1.10.0--h43eeafb_0':
+        'biocontainers/kaiju:1.10.0--h43eeafb_0' }"
+
+    input:
+    tuple val(meta), path(kaiju), path(kraken)
+    path (db)
+
+    output:
+    tuple val(meta), path("*_merged.tsv"), emit: merged
```

Comment:
```suggestion
    tuple val(meta), path("*.tsv"), emit: merged
```
The guidelines state that no hardcoded paths should be in the output files to keep the file naming as flexible as possible

Reply:
Be sure that you check that the input and output file have different names (https://nf-co.re/docs/contributing/modules#naming-conventions part 6)